client/containers/Search/Search.js

```diff
@@ -16,6 +16,18 @@ const propTypes = {
 	searchData: PropTypes.object.isRequired,
 };
 
+const getSearchPath = (query, page, mode) => {
+	const queryString = query ? `?q=${query}` : '';
```

Comment:
One alternative to this that I'd like to float is using `new URLSearchParams({...}).toString()`. We'd have to include the polyfill [here](https://github.com/pubpub/pubpub/blob/ddcfb2c2ba09e1dc700fdfb9a1ef44d0d043ab86/server/Html.js#L17) in order to support IE11, but I'd consider it a worthy addition.